tools/glfw_compile.dart

```diff
@@ -68,7 +71,14 @@ class Program {
   }
 
   static void main(List<String> args) {
+    for (var v in _requiredEnv) {
```

Comment:
Instead of failing fast, causing the user to run this multiple times, what about reporting all the missing and then failing.
var missing = [];
for(...) {
  if (!Platform...) {
    missing.add(v);
  }
}
if (missing.isNotEmpty) {
  stderr.writeln('The following environment variables are missing and must be set: ${missing.join(', ')});
  exit(1);
}